README.md

```diff
@@ -0,0 +1,12 @@
+# booster-cli-client
+
+The command line application interacting with the [booster-server](https://github.com/ArzamastsevOleksandr/booster-server).
+
+Currently, only the interactive commands are supported, meaning that to add a vocabulary entry, for example, one must separately pass the word, description, synonyms, antonyms etc, until the whole command arguments are passed.
+
+Every command can be interrupted with the `EXIT` command (`e`). Shutting down the app is also done with the `EXIT` command.
```

Comment:
Should we list other commands with descriptions or will we do it later?

Reply:
I think we should not include the whole list of commands here, at least for now, since the commands might change. 

Furthermore, we will introduce a `HELP` command soon which will give the app users all relevant information.